data/matrix_layouts/razerdefault18.json

```diff
@@ -209,5 +209,111 @@
             {"label": "\ud83e\udc3b", "matrix": [5, 16]},
             {"label": "\ud83e\udc3a", "matrix": [5, 17]}
         ]
+    },
+        "French": {
```

Comment:
Please remove 4 spaces in this row so it's aligned with the closing bracket above

Reply:
My bad fixed